outermysite/polls/imdbExactApi.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import sys
+import traceback
+from bs4 import BeautifulSoup
+
+def getMovieResults(Id):
+    
+    url = "http://www.imdb.com/title/"+Id+"/?ref_=fn_al_tt_1"
+    
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print "Unable to connect"
+        raise SystemExit("Slow internet connection")
+    
+    soup = BeautifulSoup(r.content, "html.parser")
+    
+    movieDetails = soup.find("div", {"class" : "plot_summary"})
+    
+    jsonArray = []
+    
+    if movieDetails==None:
+        print "No results found"
+        jsonObject={}
+        jsonObject['Error']="No results found"
+        return jsonObject
+    
+    description = movieDetails.find("div", {"class" : "summary_text"})
+    
+    director = movieDetails.find_all("span", {"itemprop" : "director"})
+    
+    writer = movieDetails.find_all("span", {"itemprop" : "creator"})
+    
+    actor = movieDetails.find_all("span", {"itemprop" : "actors"})
+    
+    rating = soup.find("span", {"itemprop" : "ratingValue"})
+    
+    titleDetails = soup.find("div", {"class" : "title_wrapper"})
+    
+    title = titleDetails.find("h1", {"itemprop" : "name"})
+    
+    duration = titleDetails.find("time", {"itemprop" : "duration"})
+    
+    releaseDate = titleDetails.find(
+            "meta",
+            {"itemprop" : "datePublished"}
+            )
+    
+    genre = titleDetails.find_all("span", {"itemprop" : "genre"})
+    
+    jsonObject = {}
+    
+    if not(title == None):
```

Comment:
When there is only one condition you want to do `not equal to` on. Then you should do: `title != None`